examples/gocean/eg6/Makefile

```diff
@@ -90,7 +90,7 @@ $(EXEC): $(KERNELS) $(EXTRACT_DIR)/lib_kernel_data_netcdf.a alg.o psy.o
 	$$(nf-config --flibs)
 
 transform: test.x90 extract_transform.py
-	$(PSYCLONE) -nodm -l -api "gocean1.0" -s ./extract_transform.py\
+	$(PSYCLONE) -nodm -l all -api "gocean1.0" -s ./extract_transform.py\
```

Comment:
Same comment as before. If it is needed then it should be explained in the READMEs, otherwise remove it.

Reply:
Removed.